src/pages/Option/Index.js

```diff
@@ -75,7 +75,12 @@ export function getPosition(rawPositionData,symbolData) {
       const _entryPrice = safeDiv(temp.cost,temp[2])
       // const entryPrice = ethers.utils.formatUnits(_entryPrice,0)
       const entryPrice = safeDiv2(cost,volume)
-
+      let liquidationPrice
+      if (volume >= 0) {
+        liquidationPrice = markPrice * (1 - initialMarginRatio/2) - (marginUsage-cost)/(volume*(1-initialMarginRatio/2))
```

Comment:
liquidationPrice formula needs to be revised: cost term is not reflected here. @linja19 is working on the derivation and verification. 
[details](https://www.notion.so/coinworld/futures-options-powers-4ba47625f71048068add704c0cd5e3b4#4f27cff55e45441daba1190cb866966a)

Reply:
temporarily set this to resolved, just to push the latest dev version to live